ci/requirements-py36.yml

```diff
@@ -12,6 +12,9 @@ dependencies:
   - pip
   - numba
   - bottleneck
+  - matplotlib
+  - cftime==1.0.3.4
+  - nc-time-axis
```

Comment:
Mind removing these with the notebook? or push the commits to another branch for another PR.

Reply:
Ok